### ISSUE TITLE
PKC 2.5.7 Unable to stop database to update

### ISSUE THREAD

**Issue by craZy-C** [croneter/PlexKodiConnect#578]
@croneter To save hijacking any other threads. I have created new. Since the updates of 2.5.4 I have been having errors as previously reported. Now when updating to 2.5.7 when I attempt to reset the kodi database it fails and returns with unable to stop database try again later. The logs show many many errors which may lend to a corruption from a previous failure I have had
Here is the full Kodi Log
[Full Kodi Log.txt](https://github.com/croneter/PlexKodiConnect/files/2705817/Full.Kodi.Log.txt)
At present this seems to be the only error on 2.5.7 - Not being able to reset the database, any sugestions

**Comment by croneter** [comment]
Thanks for the many logs and bug-hunting! :laughing:

**Comment by craZy-C** [comment]
I have several genre music playlists created a month ago

**Comment by starmanrf** [comment]
Oddly enough, items play while sync is active, once it it completes Kodi crashes

**Comment by croneter** [comment]
@craZy-C
Nope, will have to find a workaround. I already have something in mind

**Comment by croneter** [comment]
@starmanrf
Could you please be more specific? Is your issue related to https://github.com/croneter/PlexKodiConnect/issues/576? Logs?

**Comment by craZy-C** [comment]
@croneter Merry Christmas to you...Just to add to the issue, when you return. In regards to the playlists, when I try to play a plex created playlist it crashes kodi..There goes my Christmas playlist ;P

**Comment by craZy-C** [comment]
I am back from a quick holiday, i have discovered corrupted playlists in plex and having a nightmare removing them. they will be the cause. Once removed i will apply your update and advise

**Comment by croneter** [comment]
See here for a solution!
https://github.com/croneter/PlexKodiConnect/wiki/How-to-automatically-delete-playlists

**Comment by craZy-C** [comment]
Thankyou for that I wasn’t aware, too late for me though as I had just manually deleted the playlists all 700mb of them, will see what happens
